modules/python/python-dest.c

```diff
@@ -56,6 +57,23 @@ typedef struct
 
 /** Setters & config glue **/
 
+gchar *
+__normalize_key(const gchar *buffer)
+{
+  const gchar from = '-';
+  const gchar to = '_';
+  gchar *p;
+  gchar *normalized_key = g_strdup(buffer);
+  p = normalized_key;
+  while (*p)
+    {
+      if (*p == from)
+        *p = to;
+      p++;
+    }
+  return normalized_key;
```

Comment:
I would factor out a `str_replace_char(gchar *str, gchar from, gchar to)` function ( into `misc.c`).